setup.py

```diff
@@ -54,6 +57,15 @@ def pre_post_plugin_type():
               'avocado.plugins.init': [
                   'vt-init = avocado_vt.plugins.vt_init:VtInit',
                   ],
+              'avocado.plugins.resolver': [
+                  'vt = avocado_vt.plugins.vt_resolver:VTResolver'
+                  ],
+              'avocado.plugins.discoverer': [
+                  'vt = avocado_vt.plugins.vt_resolver:VTDiscoverer'
+                  ],
+              'avocado.plugins.runnable.runner': [
+                  'vt = avocado_vt.plugins.vt_runner:VTRunner',
```

Comment:
Is this only runnable as separate process? I wanted to run it in a more isolated environment (even more crucial for VT tests) like Podman (in my cases tried an LXC spawner) and I it seems these will only deploy the standard avocado runner. I tried temporarily fixing this to use `avocado-runner-avocado-vt` runner above but I get things like:
```
2021-05-24 18:23:37,684 lxc              L0172 INFO | Command exited with code 256 (Traceback (most recent call last):
  File "/root/avocado-runner", line 1011, in <module>
    main()
  File "/root/avocado-runner", line 1007, in main
    app.run()
  File "/root/avocado-runner", line 894, in run
    return kallable(args)
  File "/root/avocado-runner", line 983, in command_task_run
    for status in task.run():
  File "/root/avocado-runner", line 760, in run
    runner_klass = self.runnable.pick_runner_class(self.known_runners)
  File "/root/avocado-runner", line 314, in pick_runner_class
    raise ValueError('Unsupported kind of runnable: %s' % self.kind)
ValueError: Unsupported kind of runnable: avocado-vt
)
```
where it seems you should either have `avocado-vt` here so that the runner matches the runnable type or you should use `vt` as runnable type. If this is then fixes we get to here:
```
The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/avocado-runner", line 1008, in <module>
    main()
  File "/root/avocado-runner", line 1004, in main
    app.run()
  File "/root/avocado-runner", line 891, in run
    return kallable(args)
  File "/root/avocado-runner", line 980, in command_task_run
    for status in task.run():
  File "/root/avocado-runner", line 757, in run
    runner_klass = self.runnable.pick_runner_class(self.known_runners)
  File "/root/avocado-runner", line 308, in pick_runner_class
    runner = self.pick_runner_class_from_entry_point()
  File "/root/avocado-runner", line 289, in pick_runner_class_from_entry_point
    obj = ep.load()
  File "/usr/lib/python3.9/site-packages/pkg_resources/__init__.py", line 2461, in load
    return self.resolve()
  File "/usr/lib/python3.9/site-packages/pkg_resources/__init__.py", line 2471, in resolve
    raise ImportError(str(exc)) from exc
ImportError: module 'avocado_vt.plugins.vt_runner' has no attribute 'VTRunner'
)
```
which tells me that this entry point relies on `VTRunner` but the class we have defined above is `VTTestRunner`.

Am I getting this right or am I missing something here? Not sure how this entry point would have worked with the process spawner but it somehow worked with all our testing so far.

Reply:
Yes, sorry, I think I made a mistake while I was rebasing. The `VTRunner` it doesn't exist, and I will fix that, and have to investigate why is working with process spawner, thanks for pointing it out.   Just for the context. I am testing it just on process spawner for now, because there is still some problems around the setting up and cleaning the environment which have to be solved before we will be able to use different spawners.